.github/workflows/release.yaml

```diff
@@ -0,0 +1,21 @@
+name: Release Terraform Module
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: terraform-module
+          default-branch: main
+          pull-request-title-pattern: "ci: release ${version}"
```

Comment:
you will probably need the `gh pr close` and `gh pr open` if u want the checks to be retriggered

Reply:
I will continue to merge this PR and fire another one for the workaround: https://github.com/googleapis/release-please/issues/922. Otherwise, you have to re-approve this PR again. :sweat_smile: